src/javascripts/Views/recordTreeView.js

```diff
@@ -52,12 +52,47 @@ ngapp.controller('recordTreeViewController', function($scope, $element, $timeout
         e.stopImmediatePropagation();
     };
 
-    $scope.onCellMouseDown = function(index) {
+    $scope.onCellMouseDown = function(e, node, index) {
+    
```

Comment:
👾 this newline is unnecessary.  space invaders will invade other unnecessary spaces.